test/corpus/preprocessor.txt

```diff
@@ -3,13 +3,17 @@ Preprocessor include
 ========================================================================
 
 #include "foo.h"
-#include <bar.h>
+# include <bar.h>
```

Comment:
Since `<string>` is different than `"string"`, I think it would be good to test each of

- #include "string" with and without extra whitespace
- #include <string> with and without extra whitespace

Maybe keep this line as-is, and then change line 8 to `<baz.dtsi>`?

Reply:
Updated!